### ISSUE TITLE
Add configuration of trigger mode

### ISSUE THREAD

**Issue by wong2** [wong2/chat-gpt-google-extension#77]
We want to give the user the ability to choose when to trigger the ChatGPT query, possible modes:
* Always
* When query ends with question #43 
* Manually

**Comment by josStorer** [comment]
I'd prefer to keep both, support custom query symbols and manual switch
and add options_page for setting

**Comment by wong2** [comment]
I think custom symbols will be overcomplicated. Question mark is enough.

**Comment by josStorer** [comment]
some Japanese and Chinese input methods will write the question mark as a fullwidth character `？` 

also the question mark may be easily triggered by mistake in some cases, perhaps some people prefer to customize it like "show me"

but of course these may not be good enough reasons, I just think the target audience for this plugin may be quite broad, and it may be better to provide some customization ability in this area than hard coding

**Comment by wong2** [comment]
Yeah, more customization is always valuable to _some_ users, but it also increases the complexity of a product, so it's a balance.

**Comment by josStorer** [comment]
but frankly, i don't think it will increase the complexity unless we don't add any option

or we just need to replace `.endsWith("?")` to `.endsWith(options.triggerMark)`, the other parts for setting and getting options is generic

of course these are just my personal opinions

**Comment by wong2** [comment]
Related: https://github.com/wong2/chat-gpt-google-extension/issues/43
